tests/test_black.py

```diff
@@ -2014,31 +2032,6 @@ def test_symlink_out_of_root_directory(self) -> None:
             pytest.fail(f"`get_python_files_in_dir()` failed: {ve}")
         path.iterdir.assert_called_once()
         child.resolve.assert_called_once()
-        child.is_symlink.assert_called_once()
-        # `child` should behave like a strange file which resolved path is clearly
-        # outside of the `root` directory.
-        child.is_symlink.return_value = False
-        with pytest.raises(ValueError):
```

Comment:
So this is a slight behavioral change: _black_ will no longer raise an exception for these _strange files_ (e.g. Windows directory junctions), ignoring them just like symlinks. I don't know of any other _strage_ links, but I can't imagine a benefit in stopping the analysis for any of them.